lib/rspotify/playlist.rb

```diff
@@ -69,9 +69,12 @@ def self.find(user_id, id, market: nil)
     #           playlist = RSpotify::Playlist.find_by_id('00wHcTN0zQiun4xri9pmvX')
     #           playlist.class #=> RSpotify::Playlist
     #           playlist.name  #=> "Movie Soundtrack Masterpieces"
-    def self.find_by_id(id, market: nil)
+    def self.find_by_id(id, market: nil, fields: nil)
       url = "playlists/#{id}"
-      url << "?market=#{market}" if market
+      url << "?" if market || fields
+      url << "market=#{market}" if market
+      url << "&" if market && fields
```

Comment:
why do fields depend on market at all?

Reply:
fields also implies there will be multiple things. What is the value supposed to be? how does it work in the query params?